lib/manageiq/appliance_console/cli.rb

```diff
@@ -262,7 +266,7 @@ def create_key
     def install_certs
       say "creating ssl certificates"
       config = CertificateAuthority.new(
-        :hostname => host,
+        :hostname => options[:standalone] ? "localhost" : host,
```

Comment:
Why was this changed?

I think `host` is the appliance hostname, not the database host.